build.sbt

```
@@ -32,7 +32,10 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
   val excludedPackages = Seq(
     "<empty>",
-    "Reverse.*",
+    "Reverse*",
```

Some reverse controllers still show up at 0% coverage. The below suggestion should fix it:
```suggestion
    "Reverse.*",
    "controllers..*Reverse.*",
```